v6_MIGRATION_GUIDE.md

```diff
@@ -0,0 +1,226 @@
+# Braintree Android Drop-In v6 Migration Guide
+
+See the [CHANGELOG](/CHANGELOG.md) for a complete list of changes. This migration guide outlines the basics for updating your Braintree Drop-In integration from v5 to v6.
+
+v6 of the Drop-In SDK is compatible with v4 of the [Braintree Android SDK](https://github.com/braintree/braintree_android).
+
+_Documentation for v6 will be published to https://developer.paypal.com/braintree/docs once it is available for general release._
+
+## Table of Contents
+
+1. [Gradle](#gradle)
+1. [Builder Pattern](#builder-pattern)
+1. [DropInRequest](#dropinrequest)
+1. [Authorization](#authorization)
+1. [Launch Drop-In](#launch-drop-in)
+1. [Handle Drop-In Result](#handle-drop-in-result)
+1. [Fetch Last Used Payment Method](#fetch-last-used-payment-method)
+
+## Gradle
+
+Add the dependency in your `build.gradle`:
+
+```groovy
+dependencies {
+  implementation 'com.braintreepayments.api:drop-in:6.0.0'
+}
+```
+
+The credentials for accessing the Cardinal Mobile SDK have changed.
+
+Update the credentials in your top-level build.gradle to the following:
+
+```groovy
+repositories {
+    maven {
+        url "https://cardinalcommerceprod.jfrog.io/artifactory/android"
+        credentials {
+            username 'braintree_team_sdk'
+            password 'AKCp8jQcoDy2hxSWhDAUQKXLDPDx6NYRkqrgFLRc3qDrayg6rrCbJpsKKyMwaykVL8FWusJpp'
+        }
+    }
+}
+```
```

Comment:
Irrelevant of the major version bump - should these new credentials also be in our README for braintree_android?

Reply:
That might make sense yeah. They are in the [MIGRATION_GUIDE](https://github.com/braintree/braintree_android/blob/master/v4_MIGRATION_GUIDE.md#3d-secure).

Reply:
Agreed - will change that outside of this PR